projects/max14916/Makefile

```diff
@@ -0,0 +1,9 @@
+#Select the example you want to enable by choosing y for enabling and n for disabling
+DUMMY_EXAMPLE = y
```

Comment:
same comment here as for your other PR, rename `dummy` to `basic` or something else